build.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
     coroutinesVersion = '1.5.2'
     wordPressUtilsVersion = '2.4.0'
-    wordPressLoginVersion = 'trunk-a229c42dc36badcc39f7be3d3ad5231bb031eee9'
+    wordPressLoginVersion = 'trunk-6c8cd2f3c3873f3beca9309cae18c196df24eb84'
```

Comment:
See PR description for details/rationale about this choice during conflict resolution

Reply:
👋 @AliSoftware !

I understand why `trunk-6c8cd2f3c3873f3beca9309cae18c196df24eb84` was used. 💯 

Question (❓): Shouldn't we now cut a new `Login` release, or update the existing `0.11.1` draft release, to make it point to the latest commit as well, just to keep things consistent. Wdyt? 🤔

Reply:
Yes definitely, good point 👍.

Both the `release/0.11.1` branch and `0.11.1` tag were created, but time to officially publish the GitHub release and merge login lib's `release/0.11.1` back to login lib's `trunk` indeed 👍
I will try to take care of that this afternoon

Reply:
Follow-up: https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/84